### ISSUE TITLE
[FEATURE] Banner ads: differentiate between loading and showing

### ISSUE THREAD

**Issue by svprdga** [Poing-Studios/godot-admob-android#92]
**Is your feature request related to a problem? Please describe.**
Right now in order to show a banner we can call load_banner(), this method loads a banner and shows it. But the problem arises if you need to instantly render a banner, for instance you want to render a banner as soon as the player pauses the game. In that case, if the player pauses for 3-4 seconds then the banner is not likely to appear because is not loaded, it would be helpful to allow to load the banner, and then show it when necessary.

**Describe the solution you'd like**
I propose that:
- load_banner(): loads a banner ad, but does not show it
- show_banner(): shows the previously loaded banner

We should also have a signal informing us that the banner is ready to be shown.

**Describe alternatives you've considered**
Nothing yet.

**Additional context**
Interstitial and rewarded ads work like that. The native Admob libraries also differentiate between loading and showing. I think that it makes sense to keep this behaviour in this plugin

**Comment by gumaciel** [comment]
Hi @svprdga , i guess this is not totally possible because:

- When a Banner Ad is loaded, the AdMob instantly shows to the user. https://developers.google.com/admob/android/banner

But i will give a try to only show a banner if calls the `show_banner()` function. 

Anyway, will be good to have some functions like `show_banner` and `hide_banner`, some users already asked for this, we didn't put these functions before because already have a function called `destroy_banner` that "hide" the banner, and the `load_banner()` is already showing the banner.

**Comment by svprdga** [comment]
Yeah... show and hide will also work I think. The idea here is to call `show_banner` and be sure that a banner will appear instantly. It's not a critical issue though, only a nice-to-have. Thanks.
